### PR TITLE
feat: allow preview & building of subsections of the wiki

### LINKED ISSUE #31: [Task]: Improve wiki maintainers experience by allowing subsection builds only
- [ ] Allow documentation maintainers to focus on a single subsection of the wiki
   - Build
   - Get started
   - Learn
   - Maintain
- [ ] Update readme explaining new scripts 

The idea is to reduce the build time for people that maintain certain sections of the wiki, improving the overall maintainers experience

Their suggestion is to use the wiki CLI, but because everything lives on a repository, I think we can just add some scripts to the package.json so the documentation builds only specific subsections. I think this can be a collaborative work, please let me know how we are going to approach this before implementing anything

Reference original issue https://github.com/iotaledger/iota-wiki/issues/1254

### PR DESCRIPTION
**Pull request by VmMad**
# Description of change

Handle the selection of sections of the wiki by splitting the config and selecting it with an environment variable.
Add scripts to start a subsection of the wiki directly: `yarn start:section:{section}`
Only show the selected section in the navbar

## Instructions:

Without CLI:
- Run the start command with a section: `yarn start:section:get-started`
- The start time should be less than starting it all (`yarn start`)

## Links to any relevant issues

Closes #31.

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by marc2332 — APPROVED** [review]
:fire:

File: cli/src/commands/build.tsx
Comment (begonaalvarezd) [inline cli/src/commands/build.tsx:1]: Lets remove the changes of the CLI 🙏🏼

File: contentPlugins.js [hunk 1]
```diff
@@ -67,7 +69,7 @@ module.exports = async () => {
     },
   ];
 
-  const maintainPlugins = [
+  const maintainPlugin = [
```
Comment (begonaalvarezd) [inline contentPlugins.js]: why changing this from plural to singular when its an array?

File: contentPlugins.js [hunk 2]
```diff
@@ -84,20 +86,38 @@ module.exports = async () => {
     },
   ];
 
-  return [
-    {
+  const getStartedPlugin = [{
```
Comment (begonaalvarezd) [inline contentPlugins.js]: why changing this from plural to singular when its an array?

File: contentPlugins.js [hunk 3]
```diff
@@ -84,20 +86,38 @@ module.exports = async () => {
...
       id: 'get-started',
       path: path.resolve(__dirname, 'docs/get-started'),
       sidebarPath: path.resolve(__dirname, 'docs/get-started/sidebars.ts'),
       routeBasePath: 'get-started',
-    },
-    {
+  }]
+
+  const learnPlugin = [{
```
Comment (begonaalvarezd) [inline contentPlugins.js]: why changing this from plural to singular when its an array?

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Comment by begonaalvarezd** [comment]
superseeded by https://github.com/iotaledger/iota-wiki/pull/1292

**Pull request closed without merging**